examples/cpx.rs

```diff
@@ -42,7 +42,7 @@ fn main() -> ! {
     lis3dh.set_range(lis3dh::Range::G8).unwrap();
     let mut delay = Delay::new(core.SYST, &mut clocks);
 
-    let mut tracker = lis3dh.try_into_tracker().unwrap();
+    let mut tracker = Tracker::new(3700.0);
```

Comment:
I thought you removed tracker, but it's in the example here. Am I misunderstanding?